EIPS/eip-2794.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 2794
+title: Improved Mechanism for ETH 2.0 Staking
+author: Derek Zhou (@zhous), Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas)
+discussions-to: https://ethereum-magicians.org/t/improved-mechanism-for-eth-2-0-staking/4431;https://ethresear.ch/t/improved-mechanism-for-eth-2-0-staking/7695
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-07-17
+---
+
+## Simple Summary
+We propose a new staking protocol with an honor system to replace ETH 2.0's staking mechanism in which a node must stake 32 ETHs to be a validator and thus expose ETH 2.0 to extremely high risks. This proposal will bring strong and reliable security to ETH 2.0's staking.
+
+## Motivation (Risk Analysis)
+According to the latest released specifications of ETH 2.0, phase 0 needs to be activated by at least 16,384 nodes and this will lock at least 524288 ETHs till phase 1. In additon Ethereum's outshining potentials and future will undoubtedly attract more people to do this staking and cause more ETHs to be locked. Furthermore the ***ETH 2.0 Economics*** section in the official document "Ethereum Roadmap" discusses Staking Rewards from one million validators to 0ne hundred million and if this comes true a large amount of ETHs will be locked. All these factors will very likely drive ETH's price dramatically up.
```

Comment:
`0ne` should be `one`

Reply:
> This PR appears to be for saving SVG images for contracts, but the title of this PR is "Improved Mechanism for ETH 2.0 Staking". Was there perhaps a mistake? Also, I think I _just_ reviewed a very similar PR for SVG images a few minutes ago, is there a reason this is being opened again as a separate PR?

Actually there're two EIP mixed. I just don't know why I pulled a request based on a new EIP but it worked like this.